packages/frontend/src/pages/UpdateInfoPage.jsx

```diff
@@ -0,0 +1,165 @@
+import React, { useContext, useState, useEffect } from 'react'
+import { useParams, Link } from 'react-router-dom'
+import { observer } from 'mobx-react-lite'
+import { utils } from 'ethers'
+import state from '../contexts/state'
+import Header from '../components/Header'
+import Button from '../components/Button'
+import Footer from '../components/Footer'
+import './updateInfo.css'
+
+export default observer(() => {
+  const { id } = useParams()
+  const attesterId = BigInt(id).toString(10)
+  const { unirep, wallet, info } = useContext(state)
+
+  // load existing attester description to populate form
+  useEffect(() => {
+    const loadData = async () => {
+      await unirep.loadAttesterDescription(attesterId, info.network.name)
+    }
+    loadData()
+  }, [])
+
+  const attesterDesc = unirep.descriptionsByAttesterId.get(attesterId) ?? {
+    icon: '',
+    name: '',
+    description: '',
+    url: '',
+  }
+  const [icon, setIcon] = useState(attesterDesc.icon)
+  const [name, setName] = useState(attesterDesc.name)
+  const [description, setDescription] = useState(attesterDesc.description)
+  const [url, setUrl] = useState(attesterDesc.url)
+  const attesterDescHash = utils.keccak256(
+    utils.toUtf8Bytes(JSON.stringify([icon, name, description, url]))
+  )
+  const [responseMessage, setResponseMessage] = useState('')
+
+  return (
+    <div className="content">
+      <Header />
+
+      <div className="info-page-container">
+        <div>
+          <div className="info-h3">Update Attester Information</div>
+          <div className="info-p">
+            Enable users to discover your application.
+          </div>
+          <div className="signature-container">
+            <h5 style={{ marginBottom: '1rem' }}>
+              Updating the project information requires a signature from the
+              owner of the attester contract.
+            </h5>
+            <div className="info-p" style={{ fontWeight: '600' }}>
+              Hash of attester info:
+            </div>
+            <div className="message-box">{attesterDescHash}</div>
+            <Button
+              loadingText="verifying signature..."
+              onClick={async () => {
+                await wallet.load()
+                const signature = await wallet.signMessage(attesterDescHash)
+                if (signature) {
+                  const response = await unirep.updateAttesterDescription(
+                    attesterId,
```

Comment:
I think this needs to be a hexadecimal value, so `id` should work here

Reply:
fixed